### DISCUSSION TITLE
jsonSchema被动模式下多个dependencies时联动不符合预期

### DISCUSSION THREAD

**Discussion by a1945472906** [alibaba/formily#1756] [Q&A 2.x] [answered]
https://codesandbox.io/s/jjgwm?file=/src/components/test.vue

在切换select选项  checked_false 到其他选项时联动不符合预期
单纯勾选/不勾选checkBox时联动符合预期 
我认为这可能和Input2和Input3有两个dependencies有关 但查阅了文档不知道该怎么处理这个问题

**Comment by MisicDemone** [comment, marked as answer]
自定义组件编码问题，与formily无关。参考：

https://codesandbox.io/s/tender-ishizaka-eh2sh?file=/src/components/test.vue
